package.json

```diff
@@ -27,7 +27,7 @@
     "prod": "pnpm build:only && pnpm start",
     "storybook": "nuxt storybook --port=54000",
     "storybook:build": "nuxt storybook build",
-    "storybook:smoke": "pnpm storybook --ci --smoke-test",
+    "storybook:smoke": "pnpm i18n && pnpm storybook --ci --smoke-test",
```

Comment:
Why does storybook smoke test need to fetch translations?

Reply:
It's a leftover from my previous attempt at implementing this. Thank you for catching it!